code/game/jobs/job/engineering.dm

```diff
@@ -168,6 +170,7 @@
 		ACCESS_ENGINEERING_LVL3,
 		ACCESS_ENGINEERING_LVL4,
 		ACCESS_ATMOSPHERICS,
+		ACCESS_NETWORK,
```

Comment:
Explain.

Reply:
adds apc access, as shown in the bug report that i fixed

Reply:
Why do they need specific access?

Reply:
what do you mean specific access, they need apc access and i gave it to them.